tests/bench/bench.cc

```diff
@@ -308,14 +333,24 @@ int initial_load(const BenchConfig& conf,
             dd.set( 1, 3, "%.1f ops/s",
                     (double)num_sets * 1000000 / tt.getTimeUs() );
 
+            auto prefix_median_sum =
```

Comment:
Same as line 658, we can calculate this value once (outside for-loop) and use the cached value, instead of calling this every time.

Reply:
done.